lib/client/api/sharing.js

```diff
@@ -57,6 +59,60 @@ module.exports = function (jiffClient) {
    */
   jiffClient.internal_share = shareProtocol.jiff_share.bind(null, jiffClient);
 
+
+  /**
+   * Share a secret bit input
+   *
```

Comment:
this looks great - exactly how the share function should be integrated into the library. Maybe can disregard some of my above comments on the gmw_share file from the demo?